plat/kvm/Config.uk

```diff
@@ -29,6 +29,12 @@ config KVM_BOOT_PROTO_LXBOOT
 	help
 		Linux 64-bit Boot Protocol
 
+config KVM_BOOT_QEMU_VIRT
+	bool "QEMU Virt"
+	depends on KVM_VMM_QEMU && !KVM_VMM_FIRECRACKER && ARCH_ARM_64
```

Comment:
If QEMU and FC are mutually exclusive in Kconfig already, is it enough to state here `depends on ARCH_ARM_64 && KVM_VMM_QEMU`?

Reply:
I thought so too, but it would often be the case, whenever I would switch between the two, that this config would stay enabled, even though it does not make any sense. I will try to delete it again and see if it still happens.